app/person/index.js

```diff
@@ -1,21 +1,20 @@
-// NPM dependencies
-const router = require('express').Router()
+const router = require('express').Router({ mergeParams: true })
 
-// Local dependencies
+const { uuidRegex } = require('../../common/helpers/url')
 const breadcrumbs = require('../../common/middleware/breadcrumbs')
 const { PLACEHOLDER_IMAGES } = require('../../config')
 
-const { image, renderPerson } = require('./controllers')
-const { setPerson } = require('./middleware')
+const { image, personalDetails } = require('./controllers')
+const { setPerson, setBreadcrumb } = require('./middleware')
+
+router.get('/', (req, res) => res.redirect(`${req.baseUrl}/personal-details`))
```

Comment:
I wonder if we should actually remove this and just serve the `personDetails` controller at `/person/:personId`?

It's something I was considering for the new move changes actually. Would also save a further redirect and might give (a slight) performance gain?

Reply:
Interesting that it's something you've been considering removing from the new move page, I quite liked that it gave some structure to the routes where they matched the tabs. I can see both arguments though, and avoiding unnecessary redirects sounds sensible.

Reply:
Yeah, I have been in two minds about it actually. I totally agree that I liked that the URLs then matched the tab structure (especially seeing as we use a route per tab) but the other side of my head was saying "is it a bit unnecessary...?".

I think its less of a concern here as at least we don't make an API call to get the person each time but if we use `router.param` it would result in multiple calls which felt inefficient. 

Lets leave this as is for now then as it least it matches the other concept. And if we want to investigate it and see if has a big performance impact we can change both instances. At the moment neither of these are actually exposed to users as their part of the new design so it won't have an impact yet anyway.

Reply:
Added a [GH issue](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/issues/1872) for it to prompt us to pick this up separately and make a decision at some point.

Reply:
Awesome, that looks great, thanks!